node/bft/src/helpers/pending.rs

```diff
@@ -324,6 +369,53 @@ mod tests {
             assert_eq!(pending.num_sent_requests(solution_id), expected_num_sent_requests);
         }
     }
+
+    #[test]
+    fn test_expired_items() {
+        let rng = &mut TestRng::default();
+
+        // Initialize the ready queue.
+        let pending = Pending::<TransmissionID<CurrentNetwork>, ()>::new();
+
+        // Check initially empty.
+        assert!(pending.is_empty());
+        assert_eq!(pending.len(), 0);
```

Comment:
No need to check both the length and `is_empty()`, the latter checks `len == 0` in the `hashbrown` implementation (which is used internally by `std`).

Reply:
(ditto in the other tests)